tests/phpunit/integration/tests/Story_Archive.php

```diff
@@ -299,14 +307,10 @@ public function test_filter_display_post_states_custom_archive_not_published():
 			]
 		);
 
-		$actual = $this->call_private_method(
-			$this->instance,
-			'filter_display_post_states',
-			[
-				[],
-				get_post( self::$archive_page_id ),
-			]
-		);
+		$archive_page = get_post( self::$archive_page_id );
+		$this->assertNotNull( $archive_page );
+
+		$actual = $this->instance->filter_display_post_states( [], $archive_page );
```

Comment:
Oh wow, so this wasn't even a private method? 😅

Reply:
Yeah, funny that.